### ISSUE TITLE
Update react-a11y-img-has-alt - Alt-text cannot be an image file name

### ISSUE THREAD

**Issue by HamletDRC** [Microsoft/tslint-microsoft-contrib#217]
Alt-text cannot be an image file name (http://oaa-accessibility.org/wcag20/rule/27/, https://github.com/globant-ui/arialinter/blob/master/lib/rules/textAlternatives.js)
